src/java/org/apache/cassandra/service/accord/AccordCachingState.java

```diff
@@ -605,6 +605,12 @@ public boolean isCompleteable()
             return isDone();
         }
 
+        @Override
+        public V get()
```

Comment:
added for tests so we can see what we are saving to see if we should attempt to evict it.  Main eviction logic doesn't touch it while in this state